### ISSUE TITLE
testing my own images, need real images？ shouldn't it be enough to provide synthetic images and masks?

### ISSUE THREAD

**Issue by zhanghongyong123456** [zhenglab/IntrinsicHarmony#4]
![image](https://user-images.githubusercontent.com/48466610/149691344-df914ad1-193f-451e-a05a-293cf08839f1.png)
![image](https://user-images.githubusercontent.com/48466610/149692774-ec8f387f-4188-4c45-a5b0-fd1ec1bedc1c.png)
I don't quite understand why the real image is needed

**Comment by guozonghui** [comment]
During the testing phase, our model does not require a real image. 
Sorry, this error is caused by the default loading of the real image in data/ihd_dataset.py. You can simply replace the real = Image.open(target_path).convert('RGB') with real = Image.open(path).convert('RGB')  in line 103, or turning off the loading of the real image. We will update this bug as soon as possible.

**Comment by zhanghongyong123456** [comment]
> During the testing phase, our model does not require a real image. Sorry, this error is caused by the default loading of the real image in data/ihd_dataset.py. You can simply replace the real = Image.open(target_path).convert('RGB') with real = Image.open(path).convert('RGB') in line 103, or turning off the loading of the real image. We will update this bug as soon as possible.


Thank you for your prompt reply, I will modify it according to your prompt, there is another question: 
1.  I see that the output is 256x256, can I increase the resolution such as 1920 or 4k；
2. how much fps can I achieve in the test speed

**Comment by guozonghui** [comment]
Our pre-trained model may support large-resolution input and output images, but we have not tested it. Large resolution input images may need to adjust the number of layers of Encoder and Decoder for better performance. Besides, sorry, we haven't measured fps.

**Comment by zhanghongyong123456** [comment]
> Our pre-trained model may support large-resolution input and output images, but we have not tested it. Large resolution input images may need to adjust the number of layers of Encoder and Decoder for better performance. Besides, sorry, we haven't measured fps.

Thank you for your prompt reply. This model can achieve high-resolution testing. 
1.Simple operations can be achieved by modifying crop_size, right?
2. For better results, modify the number of layers of Encoder and Decoder. How many layers do you think you need to add?  Does retraining require the same large resolution images as the training set，If it's convenient, could you tell me how your dataset was made
when i test my data，this error occurs，
It should be a problem with my data, I'm not sure if my test data is correct，
The result has the output result, just no html text
![image](https://user-images.githubusercontent.com/48466610/149704059-25b460fd-ee7e-4590-9cb8-6b2a02b93178.png)

![image](https://user-images.githubusercontent.com/48466610/149703047-561a213c-c173-4fb9-95c8-9a1093a58c7d.png)

**Comment by guozonghui** [comment]
The file of the test image list may have an extra newline at the end.

**Comment by zhanghongyong123456** [comment]
> The file of the test image list may have an extra newline at the end.

yes, it is ok,
I feel that it is no different from the data set you provided, but there is a problem, it turned out to be my txt problem, thank you very much for your answer
